slides/kube/daemonset.md

```diff
@@ -458,14 +463,14 @@ The timestamps should give us a hint about how many pods are currently receiving
 
 - Remove these pods:
   ```bash
-  kubectl get pods -l run=rng,isactive!=yes -o name | xargs kubectl delete
+  kubectl delete pods -l run=rng,isactive!=yes
```

Comment:
Hmm, I think I prefer _looking_ at something first before deleting it! As do many ops folks, I suspect. Can we do both?

Reply:
Yes, that's exactly what we do with the previous command (`kubectl get pods -l run=rng,isactive!=yes`), do you think that needs to be clearer?